src/bitmessagecli.py

```diff
@@ -679,74 +702,80 @@ def attachment(): #Allows users to attach a file to their message or broadcast
             print '     ---------------------------------------------------\n'
             isImage = True
             time.sleep(2)
-            
-        print '\n     Encoding Attachment, Please Wait ...\n' #Alert the user that the encoding process may take some time.
-        
-        with open(filePath, 'rb') as f: #Begin the actual encoding
-            data = f.read(188743680) #Reads files up to 180MB, the maximum size for Bitmessage.
+
+        # Alert the user that the encoding process may take some time.
+        print '\n     Encoding Attachment, Please Wait ...\n'
+
+        with open(filePath, 'rb') as f:  # Begin the actual encoding
+            data = f.read(188743680)  # Reads files up to 180MB, the maximum size for Bitmessage.
             data = data.encode("base64")
 
-        if (isImage == True): #If it is an image, include image tags in the message
+        if isImage:  # If it is an image, include image tags in the message
             theAttachment = """
 <!-- Note: Image attachment below. Please use the right click "View HTML code ..." option to view it. -->
 <!-- Sent using Bitmessage Daemon. https://github.com/Dokument/PyBitmessage-Daemon -->
- 
-Filename:%s 
-Filesize:%sKB 
-Encoding:base64 
- 
+
+Filename:%s
+Filesize:%sKB
+Encoding:base64
+
 <center>
     <div id="image">
         <img alt = "%s" src='data:image/%s;base64, %s' />
     </div>
-</center>""" % (fileName,invSize,fileName,filetype,data)
-        else: #Else it is not an image so do not include the embedded image code.
+</center>""" % (fileName, invSize, fileName, filetype, data)
+        else:  # Else it is not an image so do not include the embedded image code.
             theAttachment = """
 <!-- Note: File attachment below. Please use a base64 decoder, or Daemon, to save it. -->
 <!-- Sent using Bitmessage Daemon. https://github.com/Dokument/PyBitmessage-Daemon -->
- 
-Filename:%s 
-Filesize:%sKB 
-Encoding:base64 
- 
-<attachment alt = "%s" src='data:file/%s;base64, %s' />""" % (fileName,invSize,fileName,fileName,data)
+
+Filename:%s
+Filesize:%sKB
+Encoding:base64
+
+<attachment alt = "%s" src='data:file/%s;base64, %s' />""" % (fileName, invSize, fileName, fileName, data)
 
         uInput = userInput('Would you like to add another attachment, (Y)es or (N)o?').lower()
 
-        if (uInput == 'y' or uInput == 'yes'):#Allows multiple attachments to be added to one message
-            theAttachmentS = str(theAttachmentS) + str(theAttachment)+ '\n\n'
-        elif (uInput == 'n' or uInput == 'no'):
+        if uInput == 'y' or uInput == 'yes':  # Allows multiple attachments to be added to one message
```

Comment:
if uInput in ['y','yes']:
     theAttachmentS = "%s%s\n\n" %(theAttachmentS, theAttachment)
elif uInput in ['n','no']:
    break